src/dehydrated_devices.rs

```diff
@@ -21,6 +21,51 @@ impl From<dehydrated_devices::DehydratedDevices> for DehydratedDevices {
     }
 }
 
+/// Dehydrated device key
+#[wasm_bindgen]
+#[derive(Debug)]
+pub struct DehydratedDeviceKey {
+    pub(crate) inner: Uint8Array,
```

Comment:
why not use `SdkDehydratedDeviceKey` here, rather than `Uint8Array`?

Reply:
Yes that's what I wanted to do intitialy. But there was an issue doing that with ffi bingings. Let me try with wasm